Journal/Journal/Views/EntryTableViewCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  EntryTableViewCell.swift
+//  Journal
+//
+//  Created by Cora Jacobson on 8/5/20.
+//  Copyright © 2020 Cora Jacobson. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+class EntryTableViewCell: UITableViewCell {
+    
+    // MARK: - Properties
+    
+    static let reuseIdentifier = "EntryCell"
+    private let formatter = DateFormatter()
```

Comment:
I think this was actually from day one (sorry I didn't say anything then) but you might consider trying to make a shared date formatter so that each cell doesn't have to create it's own. Here are a couple extensions that I made in my project to make it simple:

``` swift
extension Date {
    var shortString: String {
        return DateFormatter.shortFormatter.string(from: self)
    }
}

extension DateFormatter {
    static let shortFormatter: DateFormatter = {
        let dateFormatter = DateFormatter()
        dateFormatter.dateStyle = .short
        dateFormatter.timeStyle = .short
        return dateFormatter
    }()
}
```

Then when I want to use them it is as easy as this:

``` swift
timestampLabel.text = entry.timestamp.shortString
```